synapse/lib/cli.py

```diff
@@ -373,7 +373,7 @@ async def runCmdLoop(self):
                 if not line:
                     continue
 
-                coro = self.runCmdLine(line)
+                coro = self.runCmdLine(line, stormopts)
```

Comment:
Please specify name= syntax for any calls which specify kwargs ( for future proofing against order changes )